### PR TITLE
Youtube Data api blog

### PR DESCRIPTION
**Pull request by rana-abhishek**
Blog on using youtube data api on angular

### CODE REVIEW

File: source/_posts/angular-youtube.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+title: Youtube Data API
```
Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:2]: Title should be 55-60 character

File: source/_posts/angular-youtube.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: Meta description should be 155-160 character right now it is 195 character

Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: 1-2 long tail keyword should be there

Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: Link internally with previous content

Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: keyword 1-2 keyword should linked with Nayan website

Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: Redirect on previous blog link if possible

Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: Title is very short, it character length should be 55-60 Character. Right now it is showing only 29 Chr

File: source/_posts/angular-youtube.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+title: Youtube Data API
+date: 2020-05-21 16:36:18
+author: Abhishek Rana
+category: Web
+tags:
```
Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:6]: Tag should be 3-4 and atleast one must be long tail keyword, which is 2 to 3 word keyword

File: source/_posts/angular-youtube.md (hunk 2 above)
Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: Link to previous content which in on nayan blog or site

Comment (Rajeev111974) [inline source/_posts/angular-youtube.md:1]: Image should be optimized with Alt text

File: source/_posts/angular-youtube.md (hunk 1 above)
Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:2]: Resolved

File: source/_posts/angular-youtube.md (hunk 2 above)
Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:1]: Resolved

Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:1]: Resolved

Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:1]: Resolved

Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:1]: Resolved

Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:1]: Resolved

Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:1]: Resolved

File: source/_posts/angular-youtube.md (hunk 3 above)
Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:6]: Resolved

File: source/_posts/angular-youtube.md (hunk 2 above)
Comment (rana-abhishek) [reply source/_posts/angular-youtube.md:1]: Resolved

**Pull request merged**